README.md

```diff
@@ -482,8 +482,8 @@ Commands:
 
```

Comment:
Above, can we change the `This command is creating...` line to:

> Utility commands for using the [Cromwell](https://cromwell.readthedocs.io) workflow engine with Terra

Reply:
Fixed.